samples/CMakeLists.txt

```diff
@@ -64,14 +64,22 @@ if (CONFIG_BT_RPMSG_NRF53)
         ${NRF_DIR}/subsys/bluetooth/controller/bt_ll_softdevice_hci_rpmsg.conf
         )
     endif()
-    if (CONFIG_BOOTLOADER_MCUBOOT)
+    if (CONFIG_BOOTLOADER_MCUBOOT AND CONFIG_NRF53_UPGRADE_NETWORK_CORE)
       # Inject this configuration from parent image hci_rpmsg to enable
       # secure bootloader on the network core. This enables firmware update
       # of the network core application.
       add_overlay_config(
         hci_rpmsg
         "${ZEPHYR_NRF_MODULE_DIR}/subsys/bootloader/image/secure_boot.conf"
         )
+
+      # Inject this configuration from parent image mcuboot to enable
+      # the PCD subsystem which is used to communicate the firmware update
+      # to the network core bootloader.
+      add_overlay_config(
+        mcuboot
+        "${ZEPHYR_NRF_MODULE_DIR}/subsys/pcd/pcd.conf"
```

Comment:
why CONFIG_PCD=y is not done via `select`?

Reply:
We want the top level image to present the user with the configuration for supporting network core DFU.
However, the top level image (e.g. `peripheral_uart` does not require `CONFIG_PCD` to be set for this, but it requries MCUBoot to be compiled with `CONFIG_PCD`, hence we use this mechanism to pass the configuration from the parent image to the child image.